docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -378,6 +378,316 @@ and a file system location of ``/var/my_tables/test_table``::
+  * - ``snapshot_id``
+    - ``bigint``
+    - The identifier for the snapshot
+  * - ``parent_id``
+    - ``bigint``
+    - The identifier for the parent snapshot
+  * - ``operation``
+    - ``varchar``
+    - The type of operation performed on the Iceberg table.
+      Supported operation types in Iceberg are:
+
+      * ``append`` when new data is appended
+      * ``replace`` when files are removed and replaced without changing the data in the table
+      * ``overwrite`` when new data is added to overwrite existing data
+      * ``delete`` when data is deleted from the table  and no new data is added
+  * - ``manifest_list``
+    - ``varchar``
+    - The list of avro manifest files containing the detailed information about the snapshot changes.
+  * - ``summary``
+    - ``map(varchar, varchar)``
+    - A summary of the changes made from the previous snapshot to the current snapshot
+
+
+``$manifests`` table
+^^^^^^^^^^^^^^^^^^^^
+
+The ``$manifests`` table provides a detailed overview of the manifests
+corresponding to the snapshots performed in the log of the Iceberg table.
+
+You can retrieve the information about the manifests of the Iceberg table
+``test_table`` by using the following query::
+
+    SELECT * FROM "test_table$manifests"
+
+.. code-block:: text
+
+     path                                                                                                           | length          | partition_spec_id    | added_snapshot_id     |  added_data_files_count  | existing_data_files_count   | deleted_data_files_count    | partitions
+    ----------------------------------------------------------------------------------------------------------------+-----------------+----------------------+-----------------------+--------------------------+-----------------------------+-----------------------------+----------------------------------------------------------------------------------------------------------------------------
+     hdfs://hadoop-master:9000/user/hive/warehouse/test_table/metadata/faa19903-1455-4bb8-855a-61a1bbafbaa7-m0.avro |  6277           |   0                  | 7860805980949777961   |  1                       |   0                         |  0                          |{{contains_null=false, lower_bound=1, upper_bound=1},{contains_null=false, lower_bound=2021-01-12, upper_bound=2021-01-12}}
+
+
+The output of the query has the following columns:
+
+.. list-table:: Manifests columns
+  :widths: 30, 30, 40
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``path``
+    - ``varchar``
+    - The manifest file location
+  * - ``length``
+    - ``bigint``
+    - The manifest file length
+  * - ``partition_spec_id``
+    - ``integer``
+    - The identifier for the partition specification used to write the manifest file
+  * - ``added_snapshot_id``
+    - ``bigint``
+    - The identifier of the snapshot during which this manifest entry has been added
+  * - ``added_data_files_count``
```

Comment:
follow-up PR for exposing further fields in the $manifests table: https://github.com/trinodb/trino/pull/10809